docs/spire-multi-cluster.md

```diff
@@ -124,188 +211,42 @@ kubectl -n tornjak delete po spire-server-0
 ```console
 kubectl -n spire edit configmap spire-agent
 ```
-* make sure all the clusters point to the same Tornjak server via public
-Ingress.
-* make sure `trust_domain` is the same as Tornjak's
-* make sure the provided cluster corresponds to the values defined earlier
 
-Here is example from `space-x02` cluster:
-```
-data:
-  agent.conf: |
-    agent {
-      data_dir = "/run/spire"
-      log_level = "DEBUG"
-      server_address = "spire-server-tornjak.space-x-01-9d995c4a8c7c5f281ce13d5467ff6a94-0000.us-south.containers.appdomain.cloud"
-      server_port = "443"
-      socket_path = "/run/spire/sockets/agent.sock"
-      trust_bundle_path = "/run/spire/bundle/bundle.crt"
-      trust_domain = "openshift.space-x.com"
-    }
-    plugins {
-      NodeAttestor "k8s_psat" {
-        plugin_data {
-          cluster = "space-x02"
-        }
-      }
-```
-Restart the agents:
-```consle
-kubectl -n spire get pods
-kubectl -n spire delete pod <pod-name>
-```
-## Step 5. Configure all workload registrars
-Here we need to define format of the SVIDs created by the registrars.
-
-The easiest is to create a individual configuration files for every cluster.
-This configuration creates the following format:
-```
- "region/{{.Context.Region}}/cluster_name/{{.Context.ClusterName}}/ns/{{.Pod.Namespace}}/sa/{{.Pod.ServiceAccount}}/pod_name/{{.Pod.Name}}"
- ```
-And this references the context provided in the configuration file as well.
-The template can reference any value that is included in context map.
-Here is the list of available, self-explanatory Pod arguments:
-* Pod.Name
-* Pod.UID
-* Pod.Namespace
-* Pod.ServiceAccount
-* Pod.Hostname
-* Pod.NodeName
-
-Here is an example for `space-x02` cluster. Similar configuration applies to
-`space-x01` and `kube-01`
+Add the `NodeAttestor "aws_iid"` to the list of plugins:
 
 ```
-apiVersion: v1
-kind: ConfigMap
-metadata:
-  name: k8s-workload-registrar
-  namespace: spire
-data:
-  registrar.conf: |
-    log_level = "debug"
-    mode = "crd"
-    trust_domain = "openshift.space-x.com"
-    # enable when direct socket access to SPIRE Server available:
-    # server_socket_path = "/run/spire/sockets/registration.sock"
-    agent_socket_path = "/run/spire/sockets/agent.sock"
-    server_address = "spire-server-tornjak.space-x-01-9d995c4a8c7c5f281ce13d5467ff6a94-0000.us-south.containers.appdomain.cloud:443"
-    cluster = "space-x02"
-    # enable for label based registration:
-    # pod_label = ""
-    # enable for annotation based registration:
-    # pod_annotation = ""
-    identity_template = "region/{{.Context.Region}}/cluster_name/{{.Context.ClusterName}}/ns/{{.Pod.Namespace}}/sa/{{.Pod.ServiceAccount}}/pod_name/{{.Pod.Name}}"
-    identity_template_label = "identity_template"
-    context {
-      Region = "us-east"
-      ClusterName = "space-x02"
+plugins {
+  NodeAttestor "k8s_psat" {
+    plugin_data {
+      cluster = "cluster-name"
     }
+  }
+  NodeAttestor "aws_iid" {
+      plugin_data {}
```

Comment:
is there some sample data that should be here?

Reply:
Generally, no plugin data is needed in AWS, and the above configuration should be used. For testing or non-standard AWS environments, you may need to specify the Metadata endpoint. See more 
https://github.com/spiffe/spire/blob/main/doc/plugin_agent_nodeattestor_aws_iid.md

I added the comment.